### PR TITLE
lib: Add RT RPI 300 to supported devices

### PR DESCRIPTION
**Pull request by acostach**
Change-type: patch
Signed-off-by: Alexandru Costache <alexandru@balena.io>

This is a CDS DT, added to the Galati testbot https://dashboard.balena-cloud.com/devices/90b38fd88dc0f7e42d910b57262ab280

### CODE REVIEW

**Comment by acostach** [comment]
@balena-ci rebase

**Comment by klutchell** [comment]
Build seems to fail
```
[worker]   lib/workers/testbot.ts:68:52 - error TS1005: ';' expected.
[worker]   68  f (process.env.TESTBOT_DUT_TYPE === 'rt-rpi-300') {
[worker]                                                         ~
[worker]   lib/workers/testbot.ts:69:8 - error TS1005: ';' expected.
[worker]   69  	turn new RtRpi300(hat);
[worker]       	     ~~~
[worker]   Found 2 errors.
```

File: lib/workers/testbot.ts [hunk 1]
```diff
@@ -64,6 +65,9 @@ const resolveDeviceInteractor = (hat: TestBotHat): DeviceInteractor => {
 	if (process.env.TESTBOT_DUT_TYPE === '243390-rpi3') {
 		return new Rpi243390(hat);
 	}
+	f (process.env.TESTBOT_DUT_TYPE === 'rt-rpi-300') {
```
Comment (klutchell) [inline lib/workers/testbot.ts:68]:
```suggestion
	if (process.env.TESTBOT_DUT_TYPE === 'rt-rpi-300') {
```

**Review by klutchell — APPROVED** [review]
lgtm

**Pull request merged**